### PR TITLE
Acaldas/browser

### PR DESCRIPTION
**Pull request by acaldas**
Added support for import with ```require``` on the client side.

### CODE REVIEW

File: src/js/base/api.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-if (typeof window === "undefined" && typeof require !== "undefined") {
+if (typeof require !== "undefined") {
```
Comment (joamag) [inline src/js/base/api.js]: why this removal ?

Comment (acaldas) [reply src/js/base/api.js]: With browserify this code will run on the browser, so the window will not be undefined.

Comment (joamag) [reply src/js/base/api.js]: What about the guys that don't use browserify :) ?

Comment (joamag) [reply src/js/base/api.js]: we need a better condition

**Comment by joamag** [comment]
@acaldas any ideas on how to detect browserify ?

**Comment by acaldas** [comment]
@joamag I think the best way is to use a gulp plugin that moves the ripe.js and ripe.css files from the ```node_modules/ripe-sdk``` folder to the ```dist``` folder and include them.

**Pull request closed without merging**